src/query-builder/update-query-builder.ts

```diff
@@ -594,6 +594,69 @@ export class UpdateQueryBuilder<DB, UT extends keyof DB, TB extends keyof DB, O>
     })
   }
 
+  /**
+   * This can be used to add any additional SQL to the front of the query __after__ the `update` keyword.
```

Comment:
This is not aligned with the example, as the additional SQL is not added **right after the keyword**. Confusing.